### DISCUSSION TITLE
Disruptive (data loss) forced restart of Firefox in Linux Mint

### DISCUSSION THREAD

**Discussion by jmssil** [linuxmint/discussions#241] [Questions]
It's a Ping Pong between Mozilla and Linux Mint:

https://github.com/linuxmint/linuxmint/issues/577

https://support.mozilla.org/en-US/questions/1433187?utm_campaign=questions-reply&utm_source=notification&utm_medium=email#answer-1622676

I suppose that the only conclusion is that the problem is in the user...

**Comment by mtwebster** [comment]
You have two random internet users telling you things that don't really address anything here. The second respondent's answer brings up a good point, though: why are you updating firefox while doing some important task on it?

*Nothing* is going to update Firefox without you either having enabled automatic updates, or manually updating it with the update manager or with apt-get. Firefox self-updating is disabled in Linux Mint. If you're not using the Mint firefox package then I can't guarantee that.

**Reply by jmssil** [reply]
I used Mint's automatic updates. This Mint's feature works well for all the other packages in the system. Why shouldn't I use a feature that I like from Mint?

How is it possible that other browsers are automatically updated without causing any disruption to concurrent usage? Why is that technically impossible for Firefox?
